pkg/apis/integreatly/v1alpha1/rhmiconfig_types_test.go

```diff
@@ -0,0 +1,67 @@
+package v1alpha1
+
+import "testing"
+
+func TestValidateBackupAndMaintenance(t *testing.T) {
```

Comment:
Just a heads up: there is already an `rhmiconfig_types` test file under `test/common` here: https://github.com/integr8ly/integreatly-operator/blob/master/test/common/rhmi_config.go#L35-L110

It might not be necessary to add a new one here.

Reply:
In fact there are already some cases for overlap checks so you might be able to just add a few extra there. 

https://github.com/integr8ly/integreatly-operator/blob/master/test/common/rhmi_config.go#L54-L72

Reply:
@darahayes Those tests, any any in `test/common`, are run as part of the automated e2e test suite and are designed to be run against a real OpenShift cluster.

I think having additional unit tests that can be run locally without an OpenShift cluster can be useful for development

Reply:
Good to know!